packages/ui/src/components/DataStory/clients/WorkspaceApiClientBase.ts

```diff
@@ -21,15 +21,69 @@ export class WorkspaceApiClientBase implements WorkspaceApiClient {
 
   constructor(private transport: Transport) {
     this.initExecutionUpdateHandler();
-    this.handleNotifyObservers();
-    this.handleExecutionResult();
-    this.handleExecutionFailure();
-    this.handleUpdateStorage();
+    this.initNotifyObservers();
+    this.initExecutionResult();
+    this.initExecutionFailure();
+    this.initUpdateStorage();
     this.run = this.run.bind(this);
     this.updateDiagram = this.updateDiagram.bind(this);
   }
 
-  //<editor-fold desc="Message handler">
```

Comment:
Seen these a few times - not sure when/how introduced?

Reply:
This is a comment used for code folding, allowing us to keep the methods related to init folded.

- expanded mode
![image](https://github.com/user-attachments/assets/11b3ca5a-07ce-41ed-90e2-4e90e9dcf0be)

- collapsed mode
![image](https://github.com/user-attachments/assets/22f0710d-46bc-49d1-97b2-197ec9e0d135)